qt-models/diveplannermodel.cpp

```diff
@@ -181,7 +181,14 @@ QStringList &DivePlannerPointsModel::getGasList()
 		cylinder_t *cyl = &displayed_dive.cylinder[i];
 		if (cylinder_nodata(cyl))
 			break;
-		list.push_back(get_gas_string(cyl->gasmix));
+		/* Check if we have the same gasmix two or more times
+		 * If yes return more verbose string */
+		int same_gas = same_gasmix_cylinder(cyl, i, &displayed_dive, true);
+		if (same_gas == -1)
+			list.push_back(get_gas_string(cyl->gasmix));
+		else
+			list.push_back(get_gas_string(cyl->gasmix) + QString(" (%1 %2 ").arg(tr("cyl.")).arg(i + 1)\
```

Comment:
no need for the `\` at the end of the line. you can move the `+` from the next line back to this line.